platform-core-module/src/main/java/com/softicar/platform/core/module/CoreI18n.java

```diff
@@ -116,6 +116,8 @@
 		.de("Klasse");
 	I18n0 CLASS_NAME = new I18n0("Class Name")//
 		.de("Klassenname");
+	I18n0 CLEANS_FILE_SERVER_FROM_TEMPORARY_FILES = new I18n0("Cleans file server from temporary files")//
```

Comment:
I think should be plural, .i.e. `servers...and a period should be at the end of the sentence, IMO.

Reply:
I changed it to plural but did not add a period because it's not a full sentence. I think, the description should be as short as possible.

Reply:
well then my feeling tells me that it should start lowercase, but whatever ;)